.github/workflows/tests.yml

```diff
@@ -70,15 +70,10 @@ jobs:
           python tests/atest/run_tests.py
 
       - name: Archive acceptances test results
-        uses: actions/upload-artifact@v1.0.0
+        uses: actions/upload-artifact@v2.3.0
         with:
-          name: log-${{ matrix.python-version }}-${{ matrix.os }}
-          path: log.html
+          name: output-${{ matrix.python-version }}-${{ matrix.os }}
+          path: |
+            log.html
+            Atest.Calculator-screenshot-1.png
```

Comment:
You can have multiple screenshots. Does this support wildcards (`*`)?

Reply:
It should https://github.com/actions/upload-artifact#upload-using-a-wildcard-pattern